__test__/add_remove.test.js

```diff
@@ -1,16 +1,21 @@
-import { addElement, deleteElement } from '../src/add_remove.js';
+import { addElement, deletetask } from '../src/add_remove.js';
```

Comment:
- [ ] Good job so far 👍 but as the previous reviewer mentioned you have to test almost everything related to adding or removing a task and not only pure functions. You also have to mock the local storage and write tests to make sure the local storage is updated accordingly. You also have to mock HTML to test if add/delete functions add or remove exactly one `li` element to/from the list in the DOM as stated clearly in the requirements.
- Requirements:
![Screenshot 2022-07-14 092131](https://user-images.githubusercontent.com/85113567/178936684-4c185ede-7c52-46cc-b660-e502900c2aaa.png)

Reply:
https://www.loom.com/share/3f1a624abf2d4faf904d5f6c94b164f8